src/lambda/wizards/samDeployWizard.ts

```diff
@@ -382,6 +472,10 @@ export class DefaultSamDeployWizardContext implements SamDeployWizardContext {
                 }
             }
         })
+
+        const val = picker.verifySinglePickerOutput(pickedItem)
+
+        return (val && val.label !== bucketMessage) ? val.label : undefined
```

Comment:
This `bucketMessage` business is convoluted.

What if you made a type that was vscode.QuickPickItem & isError?: boolean, and you can check for isError on the picked item.

Reply:
✔️